scripts/create-www-vhelm.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Usage: sh create-www.sh $SUITE
```

Comment:
How is this script different from the regular helm one?

Reply:
This shouldn't be part of `main`. I will delete the file.